src/System.Threading.Timer/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      net463|AnyOS;
```

Comment:
We're going to parse it out with other properties/conditions then have other properties dependent on it.  It needs to be a property.